src/vampyr/tests/test_projector1d.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from vampyr import vampyr1d as vp
+
+def test_ScalingProjector():
+    def f(x):
+        return x
+
+    mra = vp.MultiResolutionAnalysis(box=[0, 1], order=7)
+    P_scaling = vp.ScalingProjector(mra, 2)
+    P_wavelet = vp.WaveletProjector(mra, 2)
+
+    with pytest.raises(Exception):
```

Comment:
I don't see how use `f(x) = x` will raise an exception here. Same in the next test.

Reply:
This syntax does not work in vampyr (probably should in 1d) and causes the kernel to crash (in Jupyter notebooks). Correct syntax is `f(x) = x[0]`

Reply:
Now I see it 🤦🏻 the signature is `std::function<double(const Coord<D> &r)>` so it expects one number out of it. It's not exactly great style to raise/catch `Exception` (because it's the base class of all exceptions in Python) but I think it's appropriate here, because one cannot know exactly what went wrong when calling the function.